pages/04.Reference/08.network/02.lbaas/docs.en.md

```diff
@@ -83,6 +84,14 @@ Adding a health monitor to your load balancer is optional, but we recommend it.
 
 The test protocol (health monitor type) can be configured, among other things, using the Octavia LBaaS healthmonitor resource (e.g. `openstack loadbalancer healthmonitor create`).
 
+### Pool protocols
+
+The pool together with the pool members represent a set of backend servers. It is possible to configure the protocol between backend and load balancer.
+
+Using the special PROXY protocol it is possible to retrieve the client IP address, even with TCP or non-terminated HTTPS load balancing protocols. Please refer to the HAProxy documentation for a [specification of the PROXY protocol](http://www.haproxy.org/download/1.8/doc/proxy-protocol.txt).
+
+Your application must support the PROXY protocol. It is possible to add a reverse proxy like nginx between your application and the Octavia loadbalancer to add PROXY support. The nginx documentation provides a how to for [receiving the proxy protocol](https://docs.nginx.com/nginx/admin-guide/load-balancer/using-proxy-protocol/).
```

Comment:
provides a how-to...
(add the dash in how-to)

Reply:
```suggestion
Your application must support the PROXY protocol. It is possible to add a reverse proxy like nginx between your application and the Octavia loadbalancer to add PROXY support. The nginx documentation provides a how-to for [receiving the proxy protocol](https://docs.nginx.com/nginx/admin-guide/load-balancer/using-proxy-protocol/).
```